src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -206,45 +206,26 @@ class RemoteServicesMockServer(port:Int) {
         .withStatusCode(StatusCodes.OK.intValue)
     )
 
-    val submissionPath = "/workflows/v1"
+    val submissionBatchPath = "/workflows/v1/batch"
 
     // delay for two seconds when the test asks for it
     mockServer.when(
       request()
         .withMethod("POST")
-        .withPath(submissionPath)
+        .withPath(submissionBatchPath)
         .withBody(new ParameterBody(new Parameter("workflowOptions", "two_second_delay")))
     ).respond(
       response()
         .withHeaders(jsonHeader)
         .withBody(
-          """{
-    "id": "69d1d92f-3895-4a7b-880a-82535e9a096e",
-    "status": "Submitted"
-}""")
+          """[{"id": "69d1d92f-3895-4a7b-880a-82535e9a096e", "status": "Submitted"},{"id": "69d1d92f-3895-4a7b-880a-82535e9a096f", "status": "Submitted"},{"status": "error", "message": "stuff happens"}]""")
```

Comment:
Yes. The diff is very deceptive here since it is actually new code, not really changed code - I'll update it.